epix/assetlist.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "epix",
+  "assets": [
+    {
+      "description": "The EPIX coin, the lifeblood of the Epix network, providing the utility that powers every interaction and transaction within the ecosystem.",
+      "denom_units": [
+        {
+          "denom": "aepix",
+          "exponent": 0
+        },
+        {
+          "denom": "epix",
+          "exponent": 18
+        }
+      ],
+      "base": "aepix",
```

Comment:
both this and the testnet asset require that type_asset be defined.

Reply:
Added type_asset, thanks!